lib/tasks/data_key_validation.rake

```diff
@@ -0,0 +1,52 @@
+
+namespace :data_key_validation do
+  desc "Validates the value of ENV['CONJUR_DATA_KEY']"
+  # This task validate the master data key which is stored in `ENV['CONJUR_DATA_KEY']`
+  # Once the db is created, a secret will be created using the master key.
+  # Each time the server is restarted this task will attempt to retrieve the
+  # secret, if the provided key does not match the original master the secret retrieval
+  # will fail.
```

Comment:
i would write that we have 2 cases - a first run of the server and a restart of the server. Then i would describe what this task does in each case